packages/worker/src/tests/mocks/email.ts

```diff
@@ -1,10 +1,170 @@
-export function mock() {
-  // mock the email system
-  const sendMailMock = jest.fn()
-  const nodemailer = require("nodemailer")
-  nodemailer.createTransport.mockReturnValue({
-    sendMail: sendMailMock,
-    verify: jest.fn(),
-  })
-  return sendMailMock
+import MailDev from "maildev"
```

Comment:
This is fantastic - one thing that comes to mind is that we should probably make this mail server available through `backend-core` as I suspect @melohagan will be very interested in using this in the account portal as well.

Reply:
I'm led to believe Mel is trying to reduce the account-portal's reliance on backend-core, so it's more likely a copy-paste job. Either way, I wrote it in a way that should be easily portable.